torch/_decomp/__init__.py

```diff
@@ -274,8 +274,6 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
             aten.mish_,
             aten.mse_loss,
             aten.mse_loss_backward,
-            aten.multi_margin_loss,
-            aten.multi_margin_loss_backward,
```

Comment:
@nkaretnikov you still want the decomposition for `multi_margin_loss`, it's just the `_backwards` one that shouldn't be in this list since there isn't a decomposition for it.

Reply:
Makes sense. Updated. Thanks for noticing quickly!